src/app/zap-templates/zcl/data-model/chip/thread-network-diagnostics-cluster.xml

```diff
@@ -153,9 +153,10 @@ limitations under the License.
     <attribute side="server" code="0x3B" define="SECURITY_POLICY" type="SecurityPolicy" writable="false" isNullable="true" optional="false">SecurityPolicy</attribute>
     <attribute side="server" code="0x3C" define="DIAG_CHANNEL_MASK" type="octet_string" length="4" writable="false" isNullable="true" optional="false">ChannelPage0Mask</attribute>
     <attribute side="server" code="0x3D" define="OPERATIONAL_DATASET_COMPONENTS" type="OperationalDatasetComponents" writable="false" isNullable="true" optional="false">OperationalDatasetComponents</attribute>
-    <attribute side="server" code="0x3E" define="ACTIVE_THREAD_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFaultEnum" length="4" writable="false" optional="false">ActiveNetworkFaultsList</attribute>
+    <attribute side="server" code="0x3E" define="ACTIVE_THREAD_NETWORK_FAULTS" type="ARRAY" entryType="NetworkFaultEnum" length="4" writable="false" optional="false">ActiveNetworkFaults</attribute>
```

Comment:
This is actually breaking Darwin API in a way that is not obvious for me how to fix yet.  I could try to invest a bunch of time into solving that, or we could just adjust the spec to match the SDK here...

Note that this is also breaking the java APIS in the same way, which is another reason to just change the spec.

Reply:
Ok, will create a spec PR.

Reply:
created https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7897

Reply:
Moved things back to List suffix.